### ISSUE TITLE
[IOS-Simulator] TextField Text Cursor Jumps Around After Using CMD+A then Delete to clear all text preventing further text input.

### ISSUE THREAD

**Issue by babramovitch** [JetBrains/compose-multiplatform#4787]
**Describe the bug**

When you are using a physical keyboard to type into a simulator TextField and use CMD-A to highlight everything and then delete it, the TextField begins having troubles with the cursor and text continues to get deleted as you type.

**Affected platforms**

Platform: iOS

**Versions**
- Libraries:
  - Compose Multiplatform version: 1.6.2
- Kotlin version: 1.9.23
- OS version(s) MacOS Ventura 13.6.4
- OS architecture (x86 or arm64): Arm64
- Device: Simulator iOS 17.2 iPhone 15 Pro

**To Reproduce**

1. Run this code snippet:
   ```
    fun MainViewController() =
        ComposeUIViewController {

        val value1 = remember { mutableStateOf("") }
        val value2 = remember { mutableStateOf("") }

        Column {
            Spacer(modifier = Modifier.height(200.dp))

            TextField(
                modifier = Modifier.fillMaxWidth().height(100.dp).background(Color.Gray),
                value = value1.value,
                onValueChange = {
                    value1.value = it
                }
            )

            TextField(
                modifier = Modifier.fillMaxWidth().height(100.dp).background(Color.Gray),
                value = value2.value,
                onValueChange = {
                    value2.value = it
                }
            )
        }
    }
   ```
2. Type some text into field 1
3. Type some text into field 2
4. Use CMD-A to highlight all text and field 2 and press delete
5. Continue typing in field 2 and your text continues to get overridden.

**Expected behavior**
Typing continues as normal 

**Additional Context**

Video has me typing CTRL-A but I was thinking windows.  it was CMD-A

After doing the CMD-A I'm just trying to type "hello world" in both fields.

I've tried doing a long click in the field to get the select all cursor to appear and deleting things that way and that works okay.  It appears to only be with CMD-A

I don't know if this would happen on a physical device with a Bluetooth keyboard attached.

https://github.com/JetBrains/compose-multiplatform/assets/3608443/5831e97e-32e5-4a28-a394-3c9dd12511c5

**Comment by babramovitch** [comment]
This is fixed in 1.6.10.  I'm going to close this.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-4787">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
